cmd/bucket-notification-handlers.go

```diff
@@ -142,9 +134,10 @@ func (api objectAPIHandlers) PutBucketNotificationHandler(w http.ResponseWriter,
 		if event.IsEventError(err) {
 			apiErr = toAPIError(ctx, err)
 		}
-
-		writeErrorResponse(ctx, w, apiErr, r.URL, guessIsBrowserReq(r))
-		return
+		if _, ok := err.(*event.ErrARNNotFound); !ok {
```

Comment:
Why is unavailable ARN supported for `Put Bucket Notification`?

Reply:
Previously, we would not save the notification config even if one of the target ARNs is not found. By ignoring this ARN not found error we would save the remaining ARNs atleast in the config